api/api_full.go

```diff
@@ -63,6 +63,36 @@ type FullNode interface {
 	Common
 	Net
 
+	// MethodGroup: ChainIndexer
+	// The ChainIndexer method group contains methods for interacting with the chain indexer.
+	//
+	// The `ChainValidateIndex` API serves multiple purposes:
```

Comment:
```suggestion

	// ChainValidateIndex validates and optionally backfills the chain index at a specific epoch.
	//
	// It can be used to:
```

and reword below to complete that sentence

Basically: `MethodGroup` needs to be separated, you've made it part of the same godoc, and the godoc for `ChainValidateIndex` still needs to be valid, so needs to start with "ChainValidateIndex" and should probably start with a simple summary, and then go on with detail.